scully/utils/staticServer.ts

```diff
@@ -115,7 +115,7 @@ export function closeExpress() {
 function injectReloadMiddleware(req, res, next) {
   const url = req.url;
   let path: string;
-  if (!watch) {
+  if (noWatch) {
```

Comment:
```suggestion
  if (!noWatch) {
```

Reply:
Are you sure? 
I think " !watch === noWatch"